src/lib/xapi.ts

```diff
@@ -123,14 +127,16 @@ export async function record(
 			},
 		}
 	}
-	await send(payload)
+	let resp = await send(payload)
+	return resp
```

Comment:
Can just `return await send(payload)`.

Reply:
deliberately returning the response thought it would be generally useful than just firing it off

Reply:
Oh , right you are :D